Lib/asyncio/streams.py

```diff
@@ -22,6 +23,26 @@
 _DEFAULT_LIMIT = 2 ** 16  # 64 KiB
 
 
+class StreamMode(enum.Enum):
+    READ = "read"
+    WRITE = "write"
+    READWRITE = "readwrite"
+
+    def is_read(self):
```

Comment:
I'd make these prefixed with an `_` underscore.  Exposing them as public api doesn't make a lot of sense imo.  Or maybe just make these methods top level private functions.

Reply:
Converted into private methods.